modules/cluster-logging-release-notes-5.4.z.adoc

```diff
@@ -0,0 +1,39 @@
+//Z-stream Release Notes by Version
+[id="cluster-logging-release-notes-5-4-1"]
+= Logging 5.4.1
+This release includes https://access.redhat.com/errata/RHSA-2022:2216[RHSA-2022:2216-OpenShift Logging Bug Fix Release 5.4.1].
+
+[id="openshift-logging-5-4-1-bug-fixes"]
+== Bug fixes
+* Before this update, the log file metric exporter was only reporting logs created while the exporter was running, resulting in inaccurate log growth data. This update resolves this by monitoring /var/log/pods. (https://issues.redhat.com/browse/LOG-2442[LOG-2442])
+
+* Before this update, the collector would be blocked by continually trying to use a stale connection when forwarding logs to fluentd forward receivers.  With this change, the `keepalive_timeout` value has been set to 30s so the collector will recycle the connection and retry failed messages in a more reasonable amount of time. (https://issues.redhat.com/browse/LOG-2534[LOG-2534])
```

Comment:
```suggestion
* Before this update, the collector would be blocked because it continually tried to use a stale connection when forwarding logs to fluentd forward receivers. With this release, the `keepalive_timeout` value has been set to 30 seconds (`30s`) so that the collector recycles the connection and re-attempts to send failed messages within a reasonable amount of time. (https://issues.redhat.com/browse/LOG-2534[LOG-2534])
```

Reply:
ack re 30 seconds, is there something in the style guide I missed that expresses a preference for not using retry? Retry seems more appropriate.

Reply:
> ack re 30 seconds, is there something in the style guide I missed that expresses a preference for not using retry? Retry seems more appropriate.

There's nothing wrong with `retry`, but with my edit, the phrase `and `retries to send failed messages` wouldn't be idiomatic. If you want to keep `retry`, maybe `retries sending failed messages` would work?